src/Sarif.Multitool/ValidateCommand.cs

```diff
@@ -65,12 +65,12 @@ protected override SarifValidationContext CreateContext(ValidateOptions options,
                 // Deserialize will return null if there are any JSON deserialization errors
                 // (which can happen, for example, if a property required by the schema is
                 // missing. In that case, again, there's no point in going on.
-                string sarifText = FileSystem.ReadAllText(context.TargetUri.OriginalString);
+                context.InputLogContents = FileSystem.ReadAllText(context.TargetUri.OriginalString);
 
                 PrereleaseCompatibilityTransformer.UpdateToCurrentVersion(
                     context.InputLogContents,
                     formatting: Formatting.None,
-                    out sarifText);
+                    out string sarifText);
```

Comment:
>string [](start = 24, length = 6)

I think you can just pass content.InputLogContents here and get rid of the sarifText var use.